### DISCUSSION TITLE
OUTLINE MODE IS NOT AVAILABLE IN A SEPARATE WINDOW

### DISCUSSION THREAD

**Discussion by aroy2010** [windingwind/zotero-better-notes#1215] [General]
When working on a note in a window, I am unable to enable or show the left panel to view outline, tree or mind map mode.
This problem only occurs in a separate window, as when I view the note in a tab, the left panel works fine.
Can anyone give me some guidance on how to fix this minor impasse? Please.

**Comment by windingwind** [comment]
For bugs, please report as an issue.
